File: src/ios/PrivateVar.m
/**
 * PrivateVar.m Cordova Plugin Implementation
 */

#import "PrivateVar.h"

@implementation PrivateVar

- (void)getDatas:(CDVInvokedUrlCommand*)command
{
    NSString* callbackId = command.callbackId;
    NSDictionary* datasReturn = [self getDatasToReturn];

    CDVPluginResult* pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDictionary:datasReturn];

    [self.commandDelegate sendPluginResult:pluginResult callbackId:callbackId];
}

- (NSDictionary*)getDatasToReturn
{
    /* Edit here to return data in json */
    return @{
             @"apiUrl": @"https://api.sistepar.com",
             @"apiKey": @"12345",
             };
}

@end

File: src/android/PrivateVar.java
package com.sistepar.cordova.privatevar;

import org.apache.cordova.CallbackContext;
import org.apache.cordova.CordovaPlugin;
import org.json.JSONArray;
import org.json.JSONException;
import org.json.JSONObject;

public class PrivatevarPlugin extends CordovaPlugin {
    /**
     * Ejecutar request y response a PluginResult
     * @param action          The action to execute.
     * @param args            The exec() arguments.
     * @param callback
     * @return
     * @throws JSONException
     */
    public boolean execute(String action, JSONArray args, final CallbackContext callback) throws JSONException {
        if ("get".equals(action)) {
            JSONObject json = new JSONObject();

            /* Edit here to return data in json */
            json.put("apiUrl", "https://api.sistepar.com");
            json.put("apiKey", "12345");

            callback.success(json);
			
        } else {
            return false;
        }

        return true;
    }
}
